### PR TITLE
[be] Fix translations and mistakes. Add new rules

### PR DESCRIPTION
**Pull request by PlagaMedicum**
- Updated translations
- Added some rules in grammar
- Added ignore list
- Added Specific Case rule
- Added Simple Replace rule

### CODE REVIEW

File: languagetool-language-modules/be/src/main/java/org/languagetool/rules/be/BelarusianSpecificCaseRule.java [hunk 1]
```diff
@@ -0,0 +1,115 @@
...
+import org.languagetool.*;
+import org.languagetool.rules.*;
+import org.languagetool.tools.StringTools;
+
+import java.util.*;
+
+/**
+ * A rule that matches words which need a specific upper/lowercase spelling.
+ * Based on English and Russian SpecificCaseRule
+ * @since 5.0
```
Comment (danielnaber) [inline languagetool-language-modules/be/src/main/java/org/languagetool/rules/be/BelarusianSpecificCaseRule.java:32]: Could you update this to `6.2` please?

File: languagetool-language-modules/be/src/main/java/org/languagetool/rules/be/BelarusianSpecificCaseRule.java [hunk 2]
```diff
@@ -0,0 +1,115 @@
...
+    return "BE_SPECIFIC_CASE";
+  }
+
+  @Override
+  public String getDescription() {
+    return "Напісанне спецыяльных найменняў у верхнім або ніжнім рэгістры";
+  }
+
+  @Override
+  public RuleMatch[] match(AnalyzedSentence sentence) {
```
Comment (danielnaber) [inline languagetool-language-modules/be/src/main/java/org/languagetool/rules/be/BelarusianSpecificCaseRule.java:79]: This method seems to be duplicated and only the strings seem to be changed. Could you avoid the duplication?

**Review by danielnaber — CHANGES_REQUESTED** [review]

File: languagetool-language-modules/be/src/main/java/org/languagetool/rules/be/SimpleReplaceRule.java [hunk 3]
```diff
@@ -0,0 +1,94 @@
...
+
+  public static final String BELARUSIAN_SIMPLE_REPLACE_RULE = "BE_SIMPLE_REPLACE";
+  
+  private static final Locale BE_LOCALE = new Locale("be");
+
+  public SimpleReplaceRule(ResourceBundle messages) throws IOException {
+    super(messages, new Belarusian());
+    setLocQualityIssueType(ITSIssueType.Misspelling);
+    setCategory(new Category(new CategoryId("MISC"), "Агульныя правілы"));
+  addExamplePair(Example.wrong("<marker>З большага</marker>, гэта быў добры дзень."),
```
Comment (danielnaber) [inline languagetool-language-modules/be/src/main/java/org/languagetool/rules/be/SimpleReplaceRule.java:56]: please fix the indentation here

File: languagetool-language-modules/be/src/main/resources/org/languagetool/MessagesBundle_be.properties [hunk 4]
```diff
@@ -4,30 +4,30 @@ atd = \u0412\u044b\u0437\u043d\u0430\u0447\u044b\u0446\u044c \u043c\u043e\u0432\
 ast = \u0410\u0441\u0442\u0443\u0440\u044b\u0439\u0441\u043a\u0430\u044f
```
Comment (danielnaber) [inline languagetool-language-modules/be/src/main/resources/org/languagetool/MessagesBundle_be.properties:4]: Please do not change this file in git, it needs to be edited on webtranslateit.com

**Review by danielnaber — CHANGES_REQUESTED** [review]

**Comment by PlagaMedicum** [comment]
Fixed issues above. `BelarusianSpecificCaseRule.java` was rewritten accordant with the English version.

**Comment by danielnaber** [comment]
Gtihub says "This branch cannot be rebased due to conflicts", could you update the PR?

**Comment by PlagaMedicum** [comment]
Try it now

**Comment by danielnaber** [comment]
Github still complains, see https://github.com/languagetool-org/languagetool/pull/8409

File: languagetool-language-modules/be/src/main/resources/org/languagetool/MessagesBundle_be.properties
Comment (PlagaMedicum) [inline languagetool-language-modules/be/src/main/resources/org/languagetool/MessagesBundle_be.properties:1]: It thinks that the file was modified, I guess

Comment (PlagaMedicum) [reply languagetool-language-modules/be/src/main/resources/org/languagetool/MessagesBundle_be.properties:1]: I missed a new line character. Now everything should be fine, I hope it is fine now.
If it doesn't, how can I undo my commits on this file?

Comment (danielnaber) [reply languagetool-language-modules/be/src/main/resources/org/languagetool/MessagesBundle_be.properties:1]: I've used a merge commit this time, it worked with that.

**Pull request merged**